Makefile.common

```diff
@@ -112,6 +112,7 @@ endif
 COQ_CM_LIBS=coqpp lib clib kernel library engine pretyping gramlib interp printing parsing proofs tactics vernac stm toplevel topbin tools
 ML_SOURCE_DIRS=$(addprefix $(COQ_SRC_DIR)/,$(COQ_CM_LIBS))
 ALL_ML_SOURCE_FILES=$(shell find $(ML_SOURCE_DIRS) -name '*.ml' -or -name '*.mli' -or -name '*.c' -or -name '*.h')
+DOCGRAM_SOURCE_FILES=$(shell find $(addprefix $(COQ_SRC_DIR)/, doc/tools/docgram) -name '*.ml' -or -name '*.mlg')
```

Comment:
Since there is a single source file for `doc_grammar`, would it make sense to just put it instead of this find? Or do you prefer to do like this to be more future-proof?

Reply:
Actually now that you mention it, I have a PR sitting in my disk doing a bit of refactoring on docgram and indeed it does split it in several modules, so while I didn't use the find on purpose, indeed can be useful in the future.

Reply:
Why keep these separate from ALL_ML_SOURCE_FILES?

Reply:
@ejgallego ping

Reply:
Because we shouldn't trigger a rebuild for `make world` if docgram changes as it is not in the core build graph.

[I admit the setup of the bridge is messy]